DigitalLearningSolutions.Web/ViewComponents/BackLinkViewComponent.cs

```diff
@@ -5,9 +5,13 @@
 
     public class BackLinkViewComponent : ViewComponent
     {
-        public IViewComponentResult Invoke(string aspController, string aspAction)
+        public IViewComponentResult Invoke(string aspController, string aspAction, string linkText)
         {
-            return View(new LinkViewModel(aspController, aspAction));
+            if (string.IsNullOrWhiteSpace(linkText))
+            {
+                linkText = "Go back";
+            }

```

Comment:
If the user specifies it as linkText="" then they've explicitly made the decision to have no text at the component, so overriding it can be really confusing behaviour. And the linkText will have to be specified in the HTML, because of how MVC works, so I'd be tempted to remove the default.